lib/core.js

```diff
@@ -534,24 +534,34 @@ impress.gsInterval = () => {
 };
 
 // Unload configuration and stop server
-impress.stop = () => {
+impress.stop = callback => {
   impress.cache.clear();
   const servers = impress.servers;
   if (!servers) {
     impress.log.warn('No servers active');
+    callback();
     return;
   }
+
+  let count = 0;
+  const afterClose = () => {
+    count--;
+    for (const appName in impress.applications) {
+      const application = impress.applications[appName];
+      application.emit('stop');
```

Comment:
Do we need both `stop` and `stopped` events?

Reply:
'stopped' is enough but we have 'start' and 'started' so we can remove both 'stop' and 'start' for symmetry :smile:

Reply:
Removed